microdata_validator/dataset_validator.py

```diff
@@ -134,6 +134,11 @@ def __is_data_row_consistent(temporality_type: str, data_row: tuple,
                     f"DataSet.temporalityType is {temporality_type}"
                 )
             )
+        if stop is None or str(stop).strip(" ") == "":
```

Comment:
Isn't `... or str(stop).strip() == ""` enough?
https://docs.python.org/3.8/library/stdtypes.html#str.strip

Reply:
True! Will remove! Should we add validation that requires an empty start value as well 🤔 ? (Similar to the problem we saw in 1.5)

Reply:
I'm not sure about these rules, but if you think it's a similar case to 1.5, then we probably should have it here as well.